protein_lm/modeling/models/apt/config.py

```diff
@@ -11,11 +11,32 @@ def __init__(
         position_embedding="learned",
         tokenizer=None,
         max_sequence_length = 1024,
+        use_mup = False,
+        query_zero_init = True,
+        n_layer = None,
+        initializer_range = 0.02,
+        mup_init_scale = 1.0,
+        mup_output_temp = 1.0,
+        mup_attn_mult = 1.0,
+        mup_embedding_mult = 1.0,
```

Comment:
I think this might be the result of starting to work on this implementation based on the one from Marco in GPT-NeoX and later on switching to base it on the MuP-scaling one. I will recheck them all and update accordingly.